docs/usage/security-and-permissions.md

```diff
@@ -0,0 +1,15 @@
+# Security and Permissions
+
+## Permission comparison
+
+| Permission        | Renovate hosted app |  Forking Renovate  | Why                                                           |
```

Comment:
@rarkins Do you want me to add an `email` row?

Question from user:

> I want to configure Renovate GitHub App for my organization but first, need information on why this permission is requested.
> 
> ![Screenshot 2022-02-24 at 15 21 45](https://user-images.githubusercontent.com/77614220/155667485-58ebd8b3-8ef6-405b-b0f9-2b0b484ca908.png)

@rarkins response:

> User permissions are used only if you log into the app dashboard and authorise with oauth. Ie you "sign in with GitHub" and in the process agree to the site terms, and your consent is stored against your email.
> 
> This doesn't grant (as far as I'm aware) the app to automatically gather any email addresses, and nor do we.

https://github.com/renovatebot/renovate/discussions/14405

Reply:
It's best for a different table. The email part is related to the dashboard, and it's a permission that's granted only per-user when they log in and not per-installation when it's installed.